openreferee_server/notify.py

```diff
@@ -0,0 +1,45 @@
+import json
+import threading
+
+from .operations import setup_requests_session
+
+
+class NotifyService:
+    def __init__(self, url=None, token=None) -> None:
+        self.url = url
+        self.token = token
+        self.session = None
+
+    def send(self, payload):
+        try:
+            self.session.get(self.url, data=json.dumps({"payload": payload}))
```

Comment:
Why is this a get request with a payload? Yes, technically a GET request can have a body, but it's incredibly uncommon. Use POST or similar - for *triggering* some action based on a notification this makes more sense anyway. GET should never perform an action.

Reply:
Just a mistake on my part, I will change this to a post.